### PR TITLE
add clqdr-balance-with-lp [clqdr-balance-with-lp]

### PR DESCRIPTION
**Pull request by dappguru**
Changes proposed in this pull request:
- Add clqdr-balance-with-lp strategy

### CODE REVIEW

File: src/strategies/clqdr-balance-with-lp/index.ts [hunk 1]
```diff
@@ -0,0 +1,139 @@
+import { BigNumber, BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller, call } from '../../utils';
+
+export const author = 'liquiddriver';
```
Comment (ChaituVR) [inline src/strategies/clqdr-balance-with-lp/index.ts:5]: Should be your github user name or the org username
```suggestion
export const author = 'LiquidDriver-finance';
```

File: src/strategies/clqdr-balance-with-lp/examples.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+      "0x4AcBcA6BE2f8D2540bBF4CA77E45dA0A4a095Fa2",
+      "0x4F3D348a6D09837Ae7961B1E0cEe2cc118cec777",
+      "0x6D7f23A509E212Ba7773EC1b2505d1A134f54fbe",
+      "0x07a1f6fc89223c5ebD4e4ddaE89Ac97629856A0f",
+      "0x8d5F05270da470e015b67Ab5042BDbE2D2FEFB48",
+      "0x8d07D225a769b7Af3A923481E1FdF49180e6A265",
+      "0x8f60501dE5b9b01F9EAf1214dbE1924aA97F7fd0",
+      "0x9B8e8dD9151260c21CB6D7cc59067cd8DF306D58",
+      "0x17ea92D6FfbAA1c7F6B117c1E9D0c88ABdc8b84C",
+      "0x38C0039247A31F3939baE65e953612125cB88268"
```
Comment (ChaituVR) [inline src/strategies/clqdr-balance-with-lp/examples.json:31]: Can add some addresses that can return a positive score? right now it return voting power for one address only

File: src/strategies/clqdr-balance-with-lp/index.ts [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+  const multiBeetsMaster = new Multicaller(network, provider, contractAbi, {
+    blockTag
+  });
+
+  addresses.forEach((address) =>
+    multiBeetsMaster.call(address, beetsMasterAddress, 'userInfo', ['69', address])
+  );
+  const resultBeetsMaster: Record<string, BigNumberish> =
+    await multiBeetsMaster.execute();
```
Comment (ChaituVR) [inline src/strategies/clqdr-balance-with-lp/index.ts:119]: It should not cross 5 request limit, https://github.com/snapshot-labs/snapshot-strategies#checklist-for-adding-a-new-strategy

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/strategies/clqdr-balance-with-lp/index.ts (hunk 1 above)
Comment (dappguru) [reply src/strategies/clqdr-balance-with-lp/index.ts]: Done

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**